hermes-main-website/src/app/components/header/header.component.sass

```diff
@@ -0,0 +1,87 @@
+@import 'src/variables.sass'
+
+.content-menu li
+    display: inline-block
+    background-color: $red-main
+    font-size: 25px
+    margin:0
+    margin: 5px 10px
+
+.content-menu 
+    width: 100%
+    background-color: $red-main
+    display: flex
+    justify-content: flex-end
+    padding: 0.6rem 0 0.1rem
+    
+.content-menu li a:link, a:visited, a:hover 
+    text-decoration: none
+    color: $white-main
+    
+.header-logo 
+    height: 200px
+    width: 100%
+    background-color: $white-main
+    text-align: center
+    display: flex
+    flex-direction: column
+    align-items: center
+    position: absolute
+    margin-bottom: 20px
+    padding-bottom: 20px
+    
+.header-logo img
+    height: 80%
+    width: 100%vh
+
+#button-center
```

Comment:
Button not matching the design
Please add 
 - border: none
 - padding: 3px 10px 3px 10px
 
Remove margin: 5px 10px

Also as above a little update please to font-size to 25 :D thanks :D